Make.config

```diff
@@ -113,7 +113,7 @@ WATCHOS_NUGET_VERSION_FULL=$(WATCHOS_NUGET_VERSION_NO_METADATA)+$(NUGET_BUILD_ME
 # Xcode version should have both a major and a minor version (even if the minor version is 0)
 XCODE_VERSION=11.4
 XCODE_URL=http://xamarin-storage/bot-provisioning/xcodes/Xcode_11.4.xip
-XCODE_DEVELOPER_ROOT=/Applications/Xcode114.app/Contents/Developer
+XCODE_DEVELOPER_ROOT=/Applications/Xcode_11.4.0.app/Contents/Developer
```

Comment:
bad timing
let's start with 11.5 so we don't have different branches with different requirements